library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/obfuscator/StaticStringObfuscator.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.obfuscator
+
+internal class StaticStringObfuscator : StringObfuscator {
```

Comment:
I find this name a bit confusing (I was thinking that maybe static is used in the name because it is `object` class). Maybe we can come up with a better name? Or simply use `DefaultStringObfuscator` with default value?

Reply:
Yes I have to admit the name is intriguing me also, wanted to see your opinion also. I'd rather have separated implementations for each as this might change at some point later and it is easier to joggle with unique classes. What do you think about `FixedLengthStringObfuscator` ?

Reply:
Do we need this class? We cannot use `DefaultStringObfuscator` with default value?

Reply:
I might add more strategies there so I would rather don't have a state in any of those like a `default` value.